### PR TITLE
Minor map updates.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Yoyobatty
add: Added privacy shutters for HoS, RD, CMO, CE offices and toxins.
tweak: Head office doors are now opaque instead of glass airlocks.
tweak: Atmospherics injectors for the pressure rooms have been maxed to 200L/s
tweak: Engineering lockdown blastdoors have been moved to Foyer and out of the engineering double doors right of the CE office.
tweak: Anchored Emergency O2 lockers in the airlocks between the engine and the engine room.
tweak: Atmospherics and Genetics APCs now have improved batteries.
/:cl:

I'm doing this on behalf of Yoyobatty, I'll get him to justify the changes in the comments below.

### CODE REVIEW

**Comment by KayeArray** [comment]
Likely conflicts with the latest upstream merge, sorry.

**Comment by ghost** [comment]
Did it just get merged?

**Comment by KayeArray** [comment]
Yes, hence the conflicts.

**Comment by yoyobatty** [comment]
add: Added privacy shutters for HoS, RD, CMO, CE offices and toxins.
My reasoning for this was that HOP and CE got their own privacy shutters, so why not have other heads have em?
tweak: Head office doors are now opaque instead of glass airlocks.
Again, privacy reasons to stay consistent with privacy shutters

tweak: Atmospherics injectors for the pressure rooms have been maxed to 200L/s
Less pipe clogging, and I figured why not?

tweak: Engineering lockdown blastdoors have been moved to Foyer and out of the engineering double doors right of the CE office.
Basically so when you lock down Engineering it properly locks it down.

tweak: Anchored Emergency O2 lockers in the airlocks between the engine and the engine room.
Space wind kept pulling the lockers out so they are now anchored by default.

tweak: Atmospherics and Genetics APCs now have improved batteries.
Atmospherics and Genetics APCs will die almost instantly once power goes out so it will give them a little bit longer to seek an Engineer if power isn't turned on.

**Comment by ghost** [comment]
I'll remake this PR tomorrow with the changes again instead of fighting with git.
Map merging is suffering.

**Comment by deathride58** [comment]
It should be plausible to just run the map conflict fixer instead of remaking the entire PR.
It's simple and easy to use.
